jobs/rootfs-builder.jpl

```diff
@@ -80,7 +95,14 @@ node("debos && docker") {
     def config = "${params.ROOTFS_CONFIG}"
     def arch = "${params.ROOTFS_ARCH}"
     def pipeline_version =  "${params.PIPELINE_VERSION}"
-    def docker_image = "${params.DOCKER_BASE}debos"
+    def docker_image = " "
+    def rootfs_type = "${params.ROOTFS_TYPE}"
```

Comment:
I thought ROOTFS_TYPE had been dropped, since it's abstracted by kci_rootfs?

Reply:
ROOTFS_TYPE removed from parent job `rootfs-build-trigger.jpl`. This parent job, detects the rootfs type here https://github.com/kernelci/kernelci-jenkins/commit/218204388a0efc71ec3ec287e0a39f89c1f833d5#diff-099d43d667779be48d79969ba66e1eabd51818b642a2d75cd5dff801323585c7R40
and passes the rootfs_type to child job  `rootfs-builder.jpl`. I don't think we will be invoking child job directly, so I kept the ROOTFS_TYPE parameter for child job. If we want to completely avoid ROOTFS_TYPE in both parent and child job, then we may need to re-run above listBuildrootConfigs logic into child job. 

Now looking back, probably it might be better idea to apply `listBuildrootConfigs` in child job rather than parent job. Let me move this code from parent to child job and update this PR.

Reply:
Each rootfs-builder job will be building only one rootfs image, in one Docker image.  So I don't see why you want to list the configs here.  It may be OK for now to have ROOTFS_TYPE in rootfs-builder and not rootfs-trigger, although ideally the Jenkins jobs shouldn't have any mention of debos or buildroot.  But that can be fixed in a follow-up PR.